GettingStarted/src/Samples/UserInsideEvent/SGSUserInsideEventVC.m

```diff
@@ -41,7 +41,10 @@ - (void)didReceiveMemoryWarning {
 
 - (void) startPositioning {
     [SITLocationManager sharedInstance].delegate = self;
-    SITLocationRequest *request = [[SITLocationRequest alloc] initWithPriority:1 provider:kSITInPhoneProvider updateInterval:1 buildingID:self.selectedBuildingInfo.building.identifier operationQueue:nil options:nil];
+    SITLocationRequest *request = [[SITLocationRequest alloc] init];
```

Comment:
Gustame a idea de separar a inicialización dos parametros, pareceme que queda máis claro. Sen embargo no caso da location request, houbo unhos cantos constructores que deprecamos e unhos setters que engadimos. A idea era que eses novos métodos se convertiran no modo estándar de usar a location request.

Poderías facer o mesmo que fixeches de separar a asignacion de cada parametro nunha liña, pero empregando estes metodos?
*- (instancetype) initWithBuildingId: (NSString*) buildingId;*
*- (void) setUseDeadReckoning: (BOOL) useDeadReckoning;*
*- (void) setUseGps: (BOOL) useGps;*
*- (void) setRealtimeUpdateInterval: (SITRealtimeUpdateInterval) realtimeUpdateInterval;*

Asi podemos ir movendo aos novos usuarios cara eles e pouco a pouco reducir o numero de xente usando o deprecado. (Perdoa polo comentario longuisimo) XD

Reply:
Ok, sen problema, no caso de setUseGps and setUseDeadReckoning que os poño a NO?
Entendo entón que preferides que usen os setters que a dot notation, non? Hmm, íavos dicir entón sería recomendable poñer as variables como readonly, pero claro, habería problemas de compatibilidade

Reply:
Pois a verdade que no tema da notación tes razon, non vexo porque impor unha notacion ante outra. Faleino tamén con Cris, e estiven revisando o exemplo análogo de Android e non usa realmente ningún destes parametros. Vendo que podemos manter a coherencia entre SDKs e non meternos no dilema de preferir unhha notacion ou outra (nin cambiar as interfaces, que xa é un dilema aparte XD), igual é mellor que deixemos só o `initWithBuildingId:`. Para un exemplo pode ser suficiente, e quen precise expandir un pouco máis pode explorar o que queira cos docs.

Perdoa que che faga cambialo dúas veces, mea culpa, que puiden telo mirado un pouco mais cando che deixei o comentario.

Reply:
Ok, elimino entón a priority e o realtimeUpdateInterval que estaban no constructor antigo tamén entón, non?

Nada oh.

Reply:
Todo perfecto agora. =)